server/src/models/Playlist.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+import { songSchema } from './Song';
+
+const playlistSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    user: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User',
+      required: true,
+    },
+    songs: {
```

Comment:
will you be adding songs in the same document ?

Reply:
This is currently creating new songs as a subdocument in playlist. Not the intended functionality. Will correct this to include ObjectId of song schema